apps/src/lab2/views/components/guide/Guide.tsx

```diff
@@ -24,7 +24,16 @@ const Guide: React.FunctionComponent<GuideProps> = ({
   return (
     <div
       id={id ? `${id}-container` : undefined}
-      className={modal ? styles.guideContainerModal : undefined}
+      className={
+        modal === 'gap'
+          ? classNames(
+              styles.guideContainerModal,
+              styles.guideContainerModalGap
+            )
+          : modal === 'full'
+          ? styles.guideContainerModal
+          : undefined
```

Comment:
nit: I think this could be simplified to `classNames(modal && styles.guideContainerModal, modal === 'gap' && styles.guideContainerModalGap)`